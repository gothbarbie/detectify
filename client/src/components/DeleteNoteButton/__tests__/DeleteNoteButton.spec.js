import React from 'react'
import { shallow } from 'enzyme'

import DeleteNoteButton from '../DeleteNoteButton'

describe('Components/DeleteNoteButton', () => {
  it('renders', () => {
    const props = {
      onClick: jest.fn(),
    }
    const wrapper = shallow(<DeleteNoteButton {...props} />)

    expect(wrapper).toMatchSnapshot()
  })
})
