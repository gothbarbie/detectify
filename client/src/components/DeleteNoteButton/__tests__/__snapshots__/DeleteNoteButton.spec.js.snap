// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/DeleteNoteButton renders 1`] = `
<styled.button
  onClick={[MockFunction]}
>
  <span>
    Delete Note
  </span>
   
  <Icon
    icon={
      Array [
        "fas",
        "times",
      ]
    }
  />
</styled.button>
`;
